### PR TITLE
Added "shift-X" keyboard shortcuts for analysis in Game view

### PR DESCRIPTION
**Pull request by kyleacmooney**
Quality of life improvement for in-game/after-game analysis: i.e., keyboard shortcuts

## Proposed Changes

  - Add "shift-X" shortcuts for some options in the Game view Dock and for starting/stopping Analyze mode
  - These shortcuts are...
    - "Zen mode" : Shift-z
    - "Toggle coordinates" : Shift-c
    - **"Enable/Disable AI review" : Shift-i**
    - "Analyze game" : Shift-a
    - "Review this game" : Shift-r
    - "Estimate score" : Shift-e
    - "Back to Game" : Shift-p

## Notes

  - I became a site supporter recently and am using the AI analysis a lot after my games.  I noticed I really wanted the ability to toggle it on and off easily: for each move I like to try to figure out what the correct move would have been, and then I reference AI each time.
  - Navigating my mouse to the "Disable AI review" and "Enable AI review" button in the Dock takes too much mental energy for me.
  - I checked https://ogs.readme.io/docs/reviews-and-demos#keyboard-shortcuts and also didn't find this in the open Issues on GitHub so I decided to see if I could add it myself :D 
  - I tried adding "shift-X" shortcuts for the majority of the options in the Dock, but things like "Game information", "Fork game", and "Link to game" seem to open modals which, with this simple implementation, would open on top of each other if the shortcuts were executed in succession without closing them.

## TL;DR
  - The main shortcut I am interested in is "Enable/Disable AI review" but I thought these other analysis shortcuts would be nice as well.  This is my first time making a PR to a public codebase like this–please lmk if I missed a step in the process or something!

### CODE REVIEW

**Comment by benjaminpjones** [comment]
Cool stuff, seems like these would be useful!

I just wanted to drop in to share the more up-to-date keyboard shortcut docs: https://github.com/online-go/online-go.com/wiki/Advanced-Tips-&-Tricks#keyboard-shortcuts

It's a wiki, so feel free to edit when these go in!
